examples/ct/ct_handlers_test.go

```diff
@@ -144,22 +147,28 @@ func TestPostHandlersOnlyAcceptPost(t *testing.T) {
 
 		// TODO(Martin2112): Remove this test when there are no more handlers to be implemented and
 		// rely on the handlers own tests
-		assert.Equal(t, http.StatusMethodNotAllowed, resp.StatusCode, "Wrong status code for GET to POST handler")
+		if expected, got := http.StatusMethodNotAllowed, resp.StatusCode; expected != got {
```

Comment:
Good call - I'll do that in a follow up later on, along with standardising all the `got, expected` orderings.